product/le.min.js

```diff
@@ -1,13 +1,13 @@
 /*
```

Comment:
Can I assume this whole file was re-generated?

Reply:
It should've been.
The only way it could've been affected would be if refactoring edited it.